custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/MavenHelper.java

```diff
@@ -32,9 +32,11 @@
     private static final Logger LOGGER = Logger.getLogger(MavenHelper.class.getName());
 
     private Config cfg;
+    private String mvnCommand;
```

Comment:
can be actually a singleton so that you do not need to reinit it for each `MavenHelper` instance. But it should be probably be a singleton as well so 🤷‍♂️

Reply:
Hope my current solution is the right way to do it in Java, haven't done any serious Java work for about 5 years or so.

Reply:
Let's keep it as is, can be refactored later